Modules/Loadable/InteractiveTubesToTree/Logic/vtkSlicerInteractiveTubesToTreeLogic.cxx

```diff
@@ -487,54 +510,74 @@ ::deleteTubeFromSpatialObject(vtkMRMLSpatialObjectsNode* spatialNode, std::set<i
     it = tubeIDs.find(currTubeId);
     if(it !=  tubeIDs.end())
```

Comment:
As of now VesselView code doesn't check the formatting issues on Git commit. Apparently there are many space issues in the code. I will fix them in a different commit.